collector/lib/HostInfo.cpp

```diff
@@ -177,6 +177,12 @@ bool HostInfo::IsRHEL76() {
   return collector::isRHEL76(kernel, GetOSID());
 }
 
+bool HostInfo::IsRHEL86() {
+  auto kernel = GetKernelVersion();
+  return (GetOSID() == "rhel" || GetOSID() == "rhcos") &&
+         (kernel.release.find(".el8_6.") != std::string::npos);
```

Comment:
This might be one of the rare cases in which I'd prefer an if statement over a direct return statement, think this way is a bit more readable.
```suggestion
  if (GetOSID() == "rhel" || GetOSID() == "rhcos") {
    return kernel.release.find(".el8_6.") != std::string::npos;
  }
```